### ISSUE TITLE
FE | Profile | Accessibility  | Screenreader Improvements for Nametag section

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#48263]
## Background

The Accessibility team informed us that a screenreader may not pick up the Nametag section of Profile because it is not under an H2. After considering some options, we determined that by making a coding change and including an "Landmark-role" Nametag info becomes easier to find.

Specifically this would be adding the `role='banner'` landmark role

Accessibility team does NOT suggest using aria live at all to announce this.

[Resource article](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/landmark_role) 

## Tasks
- [x] add role="banner" to nametag component
- [x] Run E2E tests covering current code and regression
- [x] All axe checks pass

## Acceptance Criteria
- [x] Complete all unit testing
- [x] End-to-end tests showing 0 violations
- [x] Notify Product Manager that the change is ready to be validated on Staging
- [ ] Incorporate any changes resulting from Staging review & re-test & have re-validated
- [ ] Confirm PM has opened a new ticket to deploy changes to Production using feature flag, if needed

## Validation
- Who can validate this ticket? (FE, BE, Design, PM)?
- How can this work be validated?
- What updates need to be made (e.g. product outline, use cases, contact center guide)

## Associated Tickets
- #47266

## How to configure this issue
- Include practice area in title -- e.g., [Design], [FE], [BE]
- Include project/initiative name - e.g., Auth Profile: Address Change Messaging
- Add label for practice area (frontend, backend, design)
- Add label for project if applicable
- Assign to a Profile team member
- Associate with an Epic if applicable
- If creating for yourself: estimate work if enough info exists to do so

**Comment by adamwhitlock1** [comment]
@SKasimow this is ready to validate. I think the easiest way to validate the fix is to inspect the page and verify that the role="banner" is present on the nametag `<section>` element.

Here is a screenshot of me validating in staging:

![Screen Shot 2022-11-15 at 8.36.26 AM.png](https://images.zenhubusercontent.com/60ddebd504ce2452d0e0fa92/476b4898-5052-4d38-82cc-93fdce869352)

**Comment by SKasimow** [comment]
@adamwhitlock1 - I validated above with user 36.

Thanks for the screenshot and directions for validating this - super helpful!
